<launch>
    <node pkg="upo_path_tracker" type="nav_node_2" name="nav_node_2" output="screen">
        <rosparam command="load" file="$(find upo_path_tracker)/cfg/navigation2.yaml"/>
        <remap from="/cmd_vel" to="/siar/nav_vel"/>
    </node>

</launch>